addons/udes_stock/models/product_template.py

```diff
@@ -14,3 +21,16 @@ class ProductTemplate(models.Model):
 
     # Default to being a stockable product
     type = fields.Selection(default="product")
+
+    u_speed_category_id = fields.Many2one(
+        comodel_name="product.category",
+        domain=_domain_speed_category,
+        string="Product Category Speed",
+        help="Product category speed to match with location speed.",
+    )
+
+    u_height = fields.Float(string="Height", help="Product height in m", default=0.0)
```

Comment:
Two suggestions:

1: The unit of measure is important when viewing the field, so I'd suggest setting the string to `Height (m)`.

2: Change the help string, I assume `m` is for metres? Best to say the explicitly in the help to be explicit.